tools/builds/Dockerfile-openstack

```diff
@@ -0,0 +1,27 @@
+FROM python:alpine
```

Comment:
Oh my. Shouldn't we use UBI?

Reply:
YES!
I want us to start providing dockerfile for running agnosticd for different usecase. Can even be per-config.


alpine is a distribution i like (small, secure, etc). I built the dockerfile and i'm using it from my laptop. That's why i started with this one, but we should have the choice, and UBI seems a good candidate.

I'm currently editing the readme to provide instruction on how to build, and how to run agnosticd using docker.